src/perlin.rs

```diff
@@ -34,22 +34,22 @@ pub fn perlin2<T: Float>(seed: &Seed, point: &math::Point2<T>) -> T {
     let frac0   = math::sub2(*point, floored);
     let frac1   = math::sub2(frac0, math::one2());
```

Comment:
Should we rename these variables to be consistent with the naming of the `surflet` params? Also applies to `perlin3` and `perlin4`.